app/src/main/java/fr/free/nrw/commons/upload/IUpload.java

```diff
@@ -0,0 +1,29 @@
+package fr.free.nrw.commons.upload;
+
+import fr.free.nrw.commons.BasePresenter;
+import java.util.List;
+
+public interface IUpload {
```

Comment:
Hi @ashishkumar468 , as we discussed privately, IClassName convention can change to ClassName convention.